pkgs/development/libraries/jcal/default.nix

```
@@ -0,0 +1,44 @@
+{ stdenv, fetchFromGitHub, autoconf, automake, which, libtool, getopt, readline}:
+
+stdenv.mkDerivation rec {
+  name = "jcal";
+  version = "v0.4.1";
+
+  src = fetchFromGitHub {
+    owner = "fzerorubigd";
+    repo = "jcal";
+    rev = "e4678bebb7efb7faa4435be3e87fc64da53e942d";
```

In order to use the tagged release, use:

```suggestion
    rev = "v${version}";
```